word.js

```diff
@@ -0,0 +1,52 @@
+// import of linked constructor file
+var Letter = require('./letter.js');
+
+
+function Word(selected) {
+// storing the string and the collection letters
+	this.selected = selected;
+	this.letterGroup = [];
+	this.found = false;
+
+// add new letters to the collection of letters
+	this.getLetter = function() {
+		for (var i = 0, i < this.selected.length, i++) {
+			this.letterGroup.push(new letter(this.selected[i]));
+		}
+	};
+
+	}
+// the current word/house is found
+	this.findWord = function() {
+		this.found = this.letterGroup.every(function(currentLetter) {
+			return currentLetter.show;
+		});
+		return this.found;
+	};
+
+// looks through letters to see if guessed letter matches
+	this.verify = function(guessLetter) {
+		var theReturn = 0;
+
+		for (var i = 0; i < this.letterGroup.length; i++) {
+			if (this.letterGroup[i].character == guessLetter) {
+				this.letterGroup[i].show = true;
+				theReturn++;
+			}
+		}
+		// if it matches letter is shown
+		return theReturn;
+	};
```

Comment:
This function I feel can be refactored in a way that will return the index numbers and then you can do the replacement as well, like so:
```javascript
return letterGroup.map(function(letter, index){
  if(letter.character == guessLetter) {return index}
})